db/migrate/20220622090856_create_recipe_foods.rb

```diff
@@ -0,0 +1,11 @@
+class CreateRecipeFoods < ActiveRecord::Migration[7.0]
+  def change
+    create_table :recipe_foods do |t|
+      t.integer :quantity
+      t.references :food, null: false, foreign_key: true
+      t.references :recipe, null: false, foreign_key: true
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
This migration should come after CreateRecipes not before, kindly change the date in the file name to something greater than the one in CreateRecipes, because it should be executed after it since it references it.

For example, you can change it to `20220622092841_create_recipe_foods.rb`, this way the migrations will be executed in the correct order.

Reply:
Also done